tests/foreman/api/test_product.py

```diff
@@ -1,52 +0,0 @@
-# -*- encoding: utf-8 -*-
```

Comment:
@omaciel That should be doable with a modest amount of work. There are older versions of the "new" API tools for creating, reading and deleting entities in the 6.0.z.1 branch.